game-headed/src/test/java/org/triplea/game/client/ui/javafx/screens/MainMenuPaneTest.java

```diff
@@ -64,6 +67,9 @@ void testInitialize() throws Exception {
 
   @AfterAll
   static void cleanup() throws Exception {
-    FieldSetter.setField(null, Application.class.getDeclaredField("userAgentStylesheet"), null);
+    final Field userAgentStyleSheetField =
+        Application.class.getDeclaredField("userAgentStylesheet");
+    userAgentStyleSheetField.setAccessible(true);
+    userAgentStyleSheetField.set(null, null);
```

Comment:
ditto on hard to grok nulling, what's the purpose to set null null?

Reply:
Again, a JavaFX hack, basically equivalent to `Application.userAgentStylesheet = null` to fix some issues with static initializers.

The only reason this test code is touched, because previously I used the internal (non-api) mockito methods to simplify this, but this caused some errorprone warnings to show up.